doc/api/parse_args.md

```diff
@@ -0,0 +1,60 @@
+# Parse arguments
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!-- source_link=lib/parse_args.js -->
+
+## `parseArgs([config])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `config` {Object} Used to provide arguments for parsing and to configure
+  the parser. `config` supports the following properties:
+  * `args` {string\[]} array of argument strings. **Default:** `process.argv`
+    with `execPath` and `filename` removed.
+  * `options` {Object} Used to describe arguments known to the parser.
+    Keys of `options` are the long names of options and values are an
+    {Object} accepting the following properties:
+    * `type` {string} Type of argument, which must be either `boolean` or `string`.
+      **Default:** `boolean`.
+    * `multiple` {boolean} Should the option accept multiple arguments.
```

Comment:
```suggestion
    * `multiple` {boolean} Whether this option can be provided multiple times. If `true`, all values will be collected in an array. If `false`, values for the option are last-wins. **Default:** `false`.
```

Also it should presumably be illegal for `type: boolean` arguments (https://github.com/pkgjs/parseargs/pull/92).

Reply:
No that is legal and somewhat useful for counting boolean options (like `-vvv`):
- https://github.com/pkgjs/parseargs/pull/80#discussion_r825359708
- https://github.com/pkgjs/parseargs/pull/92

Reply:
Resolved partially by: https://github.com/pkgjs/parseargs/pull/93